frame/contracts/src/lib.rs

```diff
@@ -137,7 +139,7 @@ pub use weights::WeightInfo;
 
 pub use crate::{
 	address::{AddressGenerator, DefaultAddressGenerator},
-	exec::Frame,
+	exec::{ExportedFunction, Frame},
```

Comment:
this is not needed right?

Reply:
Re-export from the `unsafe_debug` module then.